default.nix

```diff
@@ -45,6 +46,27 @@ let
           "--ghc-options=-DCONFIG=${dconfig}"
           "--ghc-options=-DGITREV=${gitrev}"
         ];
+      } // optionalAttrs (genesis != null) {
```

Comment:
FYI: this will be simplified with https://github.com/input-output-hk/cardano-sl/pull/1463